IOPool/Streamer/test/RunSimple_NewStreamer.sh

```diff
@@ -31,12 +31,14 @@ cd ${OUTDIR}
 cmsRun NewStreamOut_cfg.py compAlgo=${TEST_COMPRESSION_ALGO} > out 2>&1 || die "cmsRun NewStreamOut_cfg.py compAlgo=${TEST_COMPRESSION_ALGO}" $?
 cmsRun NewStreamOutAlt_cfg.py compAlgo=${TEST_COMPRESSION_ALGO} > outAlt 2>&1 || die "cmsRun NewStreamOut_cfg.py compAlgo=${TEST_COMPRESSION_ALGO}" $?
 cmsRun NewStreamOutExt_cfg.py compAlgo=${TEST_COMPRESSION_ALGO} > outExt 2>&1 || die "cmsRun NewStreamOut_cfg.py compAlgo=${TEST_COMPRESSION_ALGO}" $?
+cmsRun NewStreamOutExt2_cfg.py compAlgo=${TEST_COMPRESSION_ALGO} > outExt 2>&1 || die "cmsRun NewStreamOut_cfg.py compAlgo=${TEST_COMPRESSION_ALGO}" $?
```

Comment:
```suggestion
cmsRun NewStreamOutExt2_cfg.py compAlgo=${TEST_COMPRESSION_ALGO} > outExt 2>&1 || die "cmsRun NewStreamOutExt2_cfg.py compAlgo=${TEST_COMPRESSION_ALGO}" $?
```
(if you feel like it, `Alt` and `Ext` could be added to the two lines above)

How about also adding a check that the `CHECKSUM`  printout is as expected? (I see also this check is missing for the `Alt` and `Ext` tests)

Reply:
Alt and Ext combine multiple input files, so the check can not be done (it is comparing products between a single input or output process).
Anyway, streamer reader checks the header and each event for adler32 checksum consistency.